packages/react-components/react-search/library/src/components/SearchBox/useSearchBoxStyles.styles.ts

```diff
@@ -74,10 +74,10 @@ const useContentAfterStyles = makeStyles({
     columnGap: tokens.spacingHorizontalXS,
   },
   rest: {
-    opacity: 0,
     height: 0,
     width: 0,
     paddingLeft: 0,
+    overflow: 'hidden',
```

Comment:
**question:** curious why using overflow instead of previously  used visibility: hidden ?

Reply:
They are similar, overflow has the same behavior and is also accessibility compliant